dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.2_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -36,3 +37,24 @@ d//
 delimiter ;
 CALL uc_dolphin_T_t_ds_process_instance_A_restart_time();
 DROP PROCEDURE uc_dolphin_T_t_ds_process_instance_A_restart_time;
+
+
+-- uc_dolphin_T_t_ds_process_task_relation_A_pc_pd_index
```

Comment:
Should we change the 2.0.2 version?

Reply:
> * Make sql upgrade same as release version

This makes our updated SQL the same as the released version. I think it is because we have some commit directly to `2.0.x-prepare` branch and forget to migrate the code to dev branch